src/js/controller/controller.js

```diff
@@ -254,6 +255,17 @@ Object.assign(Controller.prototype, {
                 // this player has been destroyed
                 return;
             }
+
+            // Detect and store browser autoplay setting in the model.
+            const adConfig = _this._model.get('advertising');
+            canAutoplay(mediaPool, {
+                cancelable: checkAutoStartCancelable,
+                muted: _this.getMute(),
+                allowMuted: adConfig ? adConfig.autoplayadsmuted : false
```

Comment:
If we want to test a muted video tag when the unmuted test fails this should go `allowMuted: adConfig ? adConfig.autoplayadsmuted : true` correct?  The player should perform both tests because the current player behavior (on mobile) is to autostart muted if that is the only way autostart works.

Reply:
On mobile, I don't think we ever attempt to autoplay unmuted - `_this.getMute()` will return `true` if `autostartMuted` or already muted. `autostartMuted` is set if we want to autostart on mobile - hence we only do one test (unmuted) on mobile.

Reply:
Looks like you are right here, if we want to duplicate the mobile behavior (play silently unless `ads && !autoplayAdsMuted`), we should default to `true`.